packages/react-stand-in/src/utils.js

```diff
@@ -78,7 +78,7 @@ const ES5ProxyComponentFactory = function(
   return ProxyComponent
 }
 
-export const isReactIndeterminateResult = el =>
+export const isReactComponentInstance = el =>
```

Comment:
This is a slightly clearer name, I think, since it's what we're actually checking for here.